challenges/classes.js

```diff
@@ -1,7 +1,48 @@
 // 1. Copy and paste your prototype in here and refactor into class syntax.
 
+// function CuboidMaker (attr) {
+//     this.length = attr.length;
+//     this.width = attr.width;
+//     this.height = attr.height;
+//   }
+//   CuboidMaker.prototype.volume = function () {
+//     return this.length * this.width * this.height
+//   }
+//   CuboidMaker.prototype.surfaceArea = function () {
+//     return (2 * (this.length * this.width + this.length * this.height + this.width * this.height))
+//   }
+  
+  class CuboidMaker {
+      constructor ({length, width, height}){
+      this.length = length;
+      this.width = width;
+      this.height = height;
```

Comment:
Nice use of destructuring here to keep the function readable and understandable.